### DISCUSSION TITLE
Is there a way to make all minecarts, not just minecarts with entities run at high speeds?

### DISCUSSION THREAD

**Discussion by Landram5** [audaki/minecraft-cart-engine#25] [Q&A]
I want to do a ticketing system and that requires a chest minecart to run in front of the player but that seems to tank the player's speed.

**Comment by audaki** [comment]
@Landram5 Yes, the chest minecart is not made faster, because that would break redstone (like autosorters). And therefore the chest minecart will slow down the player’s minecart.

It would be possible to also make this mod apply to chest minecarts, but that would then break redstone.

So there is no easy solution.

Feel free to fork this mod and create your own version where it also makes chest minecarts faster!
